odd-platform-api/src/main/resources/application.yml

```diff
@@ -26,6 +26,39 @@ auth:
   ingestion:
     filter:
       enabled: false
+
+#  oauth2:
```

Comment:
@adyachenkoprov @damirabdul
gents, let's add new provider docs to the GitBook right away, if needed

Reply:
@DementevNikita Ok. I'll do